app/src/components/slidesContainer.js

```diff
@@ -17,50 +17,59 @@ export class SlidesContainer extends Component {
 
     this.slides = [...this.element.querySelectorAll(".slide")];
     this.prefersReducedMotion = false;
-    this.activeSlide = this.store.getState().slides.curIndex;
+    this.activeSlideIdx = this.store.getState().slides.curIndex;
 
     this.handleSlidesUpdate = this.handleSlidesUpdate.bind(this);
     this.scrollToActiveSlide = this.scrollToActiveSlide.bind(this);
     this.handleMotionQuery = this.handleMotionQuery.bind(this);
+    this.handleScrollComplete = this.handleScrollComplete.bind(this);
 
     this.handleMotionQuery();
     this.unsubscribe = observeStore(
       this.store,
       (state) => state.slides,
       this.handleSlidesUpdate
     );
+    this.scrollToActiveSlide();
   }
 
   handleSlidesUpdate() {
     const { slides } = this.store.getState();
     if (slides.curIndex !== this.activeSlideIdx) {
+      this.previousSlideIndex = this.activeSlideIdx;
```

Comment:
remove this line unless attempting to use `gsap.fromTo()` instead of `gsap.to()`